src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -855,6 +862,20 @@ public SauceConnectHandler(
             this.verboseLogging = sauceOnDemandBuildWrapper.isVerboseLogging();
             this.sauceConnectPath = sauceOnDemandBuildWrapper.getSauceConnectPath();
             this.sauceConnectJar = sauceConnectJar;
+            try {
+                this.maxRetries = Integer.parseInt(maxRetries);
+            } catch (NumberFormatException e) {
```

Comment:
will this also raise null pointer exception, since by default maxRetries is null ?